src/solver/optimisation/opt_fonctions.h

```diff
@@ -129,6 +129,15 @@ void OPT_InitialiserLeSecondMembreDuProblemeLineaireCoutsDeDemarrage(PROBLEME_HE
 void OPT_DecompteDesVariablesEtDesContraintesCoutsDeDemarrage(PROBLEME_HEBDO*);
 void OPT_InitialiserNombreMinEtMaxDeGroupesCoutsDeDemarrage(PROBLEME_HEBDO*);
 void OPT_AjusterLeNombreMinDeGroupesDemarresCoutsDeDemarrage(PROBLEME_HEBDO*);
+
+void OPT_ConstruireLaMatriceDesContraintesDuProblemeLineaireRampesThermiques(PROBLEME_HEBDO*, bool);
+void OPT_ConstruireLaListeDesVariablesOptimiseesDuProblemeLineaireRampesThermiques(PROBLEME_HEBDO*, bool);
```

Comment:
it is worthed to name parameters ;)

Reply:
resolved in commit [e78c034](https://github.com/AntaresSimulatorTeam/Antares_Simulator/pull/1817/commits/e78c03491ac707703a8f888f11e1fce0270a6560)